King-Benx/laboremus-challenge
# create home screen

**Overview**
The manager should be able to see the list of farmers.

**Description**
Once a manager has successfully signed up, they should be able to see the list of farmers.

**How to test**
1. Clone the project `git clone git@github.com:King-Benx/laboremus-challenge.git`
2. Checkout to ft-home-screen branch `git checkout ft-home-screen`
3. Install dependencies `yarn`
4. Run the development server `react-native start`
5. Run the application `react-native run-android`
6. Observe as you move from the welcome to the home screen
